### PR TITLE
CDAP-8334 Tephra state table should be created in system namespace

### PR DESCRIPTION
**Pull request by gokulavasan**
JIRA : https://issues.cask.co/browse/CDAP-8334

Build : http://builds.cask.co/browse/CDAP-RUT582

### CODE REVIEW

File: cdap-common/src/main/resources/cdap-default.xml [hunk 1]
```diff
@@ -664,22 +664,27 @@
...
+    <name>data.tx.pruning.plugin.class</name>
     <value>co.cask.data2.txprune.DefaultHBaseTransactionPruningPlugin</value>
     <description>
       Class name for the default transaction pruning plugin
     </description>
   </property>
 
   <property>
+    <name>data.tx.prune.state.table</name>
+    <value>${root.namespace}_system:tephra.state</value>
```
Comment (gokulavasan) [inline cdap-common/src/main/resources/cdap-default.xml]: Is the name 'system' for system namespace configurable? I am assuming its not in this change.

Comment (poornachandra) [reply cdap-common/src/main/resources/cdap-default.xml]: I see that `system` is hardcoded in `NamespaceId` class.

**Comment by poornachandra** [comment]
LGTM

**Comment by poornachandra** [comment]
LGTM

File: cdap-common/src/main/resources/cdap-default.xml [hunk 2]
```diff
@@ -664,22 +664,27 @@
...
+    <name>data.tx.pruning.plugin.class</name>
     <value>co.cask.data2.txprune.DefaultHBaseTransactionPruningPlugin</value>
     <description>
       Class name for the default transaction pruning plugin
     </description>
   </property>
 
   <property>
+    <name>data.tx.prune.state.table</name>
+    <value>${dataset.table.prefix}_system:tephra.state</value>
```
Comment (anew) [inline cdap-common/src/main/resources/cdap-default.xml:684]: isn't this a namespace? Why use the table prefix?

**Pull request merged**